cli/root_test.go

```diff
@@ -111,21 +127,33 @@ ExtractCommandPathsLoop:
 			}
 			err := os.Chdir(tmpwd)
 			var buf bytes.Buffer
-			root, _ := clitest.New(t, tt.cmd...)
-			root.SetOut(&buf)
+			cmd, cfg := clitest.New(t, tt.cmd...)
+			clitest.SetupConfig(t, rootClient, cfg)
+			cmd.SetOut(&buf)
 			assert.NoError(t, err)
-			err = root.ExecuteContext(ctx)
+			err = cmd.ExecuteContext(ctx)
 			err2 := os.Chdir(wd)
 			require.NoError(t, err)
 			require.NoError(t, err2)
 
 			got := buf.Bytes()
-			// Remove CRLF newlines (Windows).
-			got = bytes.ReplaceAll(got, []byte{'\r', '\n'}, []byte{'\n'})
 
-			// The `coder templates create --help` command prints the path
-			// to the working directory (--directory flag default value).
-			got = bytes.ReplaceAll(got, []byte(fmt.Sprintf("%q", tmpwd)), []byte("\"[current directory]\""))
+			replace := map[string][]byte{
+				// Remove CRLF newlines (Windows).
+				string([]byte{'\r', '\n'}): []byte("\n"),
+				// The `coder templates create --help` command prints the path
+				// to the working directory (--directory flag default value).
+				fmt.Sprintf("%q", tmpwd): []byte("\"[current directory]\""),
+			}
+			for k, v := range replacements {
+				replace[k] = []byte(v)
+			}
+			for k, v := range replace {
+				got = bytes.ReplaceAll(got, []byte(k), v)
```

Comment:
Maybe we can just use a `map[string]string` and `strings.ReplaceAll` to simplify and avoid the byte/string conversions?

Reply:
We'd have to convert `got` to a string in that case:
```go
gotStr := strings.ReplaceAll([]byte(got), k, v)
got = []byte(gotStr)
```